test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -1604,6 +1604,19 @@ describe('Dropdown', () => {
       dropdownMenuIsOpen()
     })
 
+    it('Don\'t open the menu on change if query\'s length is less than minCharacters', () => {
```

Comment:
When a string includes single quote characters, please quote it with double quotes to prevent escaping.